src/utils/urlUtils.ts

```diff
@@ -38,6 +38,20 @@ export function getDataFile(): string {
   }
   return data;
 }
+export function isItLastAssessment(): boolean {
+
+  let pathParams = getPathName();
+  let requiredScore = Number(pathParams.get("requiredScore"));
+  return requiredScore >= 640
```

Comment:
Let's remove the isItLastAssessment routine, as it's not being used anymore? @amitsinghsutara